tbears/libs/icon_integrate_test.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 ICON Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""IconServiceEngine testcase
+"""
+
+import hashlib
+import random
+from shutil import rmtree
+import sys
+from time import time, sleep
+from typing import Any
+from unittest import TestCase
+
+from iconcommons import IconConfig
+from iconsdk.converter import convert_transaction_result
+from iconsdk.exception import IconServiceBaseException
+from iconsdk.icon_service import IconService
+from iconsdk.builder.call_builder import Call
+from iconsdk.signed_transaction import SignedTransaction
+from iconservice.base.address import Address
+from iconservice.base.block import Block
+from iconservice.base.type_converter import TypeConverter, ParamType
+from iconservice.icon_config import default_icon_config
+from iconservice.icon_constant import ConfigKey, DATA_BYTE_ORDER
+from iconservice.icon_service_engine import IconServiceEngine
+from iconservice.icon_inner_service import MakeResponse
+from iconservice.utils import to_camel_case
+from iconsdk.wallet.wallet import KeyWallet
+from tbears.config.tbears_config import TEST1_PRIVATE_KEY
+
+SCORE_INSTALL_ADDRESS = f"cx{'0'*40}"
+
+
+def create_hash_256(data: bytes=None) -> bytes:
+    if data is None:
+        max_int = sys.maxsize
+        length = (max_int.bit_length() + 7) // 8
+        data = int(random.randint(0, max_int)).to_bytes(length, DATA_BYTE_ORDER)
+
+    return hashlib.sha3_256(data).digest()
+
+
+def create_tx_hash(data: bytes=None) -> bytes:
+    return create_hash_256(data)
+
+
+def create_block_hash(data: bytes=None) -> bytes:
+    return create_tx_hash(data)
+
+
+def root_clear(score_path: str, state_db_path: str):
+    rmtree(score_path, ignore_errors=True)
+    rmtree(state_db_path, ignore_errors=True)
+
+
+def create_timestamp():
+    return int(time() * 10 ** 6)
+
+
+class IconIntegrateTestBase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._score_root_path = '.testscore'
+        cls._state_db_root_path = '.teststatedb'
+
+        cls._icx_factor = 10 ** 18
+
+        cls._genesis: 'KeyWallet' = KeyWallet.create()
+        cls._fee_treasury: 'KeyWallet' = KeyWallet.create()
+        cls._test1: 'KeyWallet' = KeyWallet.load(bytes.fromhex(TEST1_PRIVATE_KEY))
```

Comment:
I think that it good to be mentioned about this in docs 
or should be added example like using _test1 and _wallet_array